pkg/volume/util_test.go

```diff
@@ -24,106 +24,174 @@ import (
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/errors"
 	"k8s.io/kubernetes/pkg/api/resource"
+	"k8s.io/kubernetes/pkg/watch"
 )
 
-func TestRecyclerSuccess(t *testing.T) {
-	client := &mockRecyclerClient{}
-	recycler := &api.Pod{
-		ObjectMeta: api.ObjectMeta{
-			Namespace: api.NamespaceDefault,
-		},
-		Status: api.PodStatus{
-			Phase: api.PodSucceeded,
-		},
-	}
+type testcase struct {
+	name          string
+	existingPod   *api.Pod
+	createPod     *api.Pod
+	eventSequence []watch.Event
```

Comment:
The difference between `eventSequence` and `expectedEvents` is not clear to me here